website/content/api-docs/system/internal-specs-openapi.mdx

```diff
@@ -31,6 +31,10 @@ This endpoint returns a single OpenAPI document describing all paths visible to
 | :----- | :---------------------------- |
 | `GET`  | `/sys/internal/specs/openapi` |
 
+### Parameters
+
+- `generic_mount_paths` `(bool: false)` – Used to specify whether to use a generic mount path. If set, the mount path will be replaced with a dynamic parameter: `{mountPath}`
```

Comment:
```suggestion
- `generic_mount_paths` `(bool: false)` – Used to specify whether to use a generic mount paths. If set, the mount paths will be replaced with a dynamic parameter: `{mountPath}`
```

It may be good to include an example here as well

Reply:
done